src/Solver.h

```diff
@@ -126,9 +126,9 @@ class Solver {
   /** The total number of Tracks */
   int _tot_num_tracks;
 
-  /** An integer array with the Track uid separating the azimuthal and periodic
-   * halfspaces */
-  int* _num_tracks_by_halfspace;
+  /** An integer array with the Track uid separating the parallel track
```

Comment:
It is not really needed here, but the fact that the Solver's `_tracks` array is arranged so the Track UID corresponds to its index in the Solver's `_tracks` array is not coincidence as the UID is used as the index in the `dev_tracks` array in the `GPUSolver::initializeTracks` method. I'll remove the mention of uid here.